app/views/pages/all_users.html.erb

```diff
@@ -14,6 +14,11 @@
         <td>
           <%= u.username %>
         </td>
+        <% if policy(u).destroy? %>
+          <td>
+            <%= link_to 'Destroy', u, method: :delete, data: { confirm: 'Are you sure?' } %>
+          </td>
+        <% end %>
```

Comment:
extra point if you can refactor this logic using locals :D
https://guides.rubyonrails.org/layouts_and_rendering.html#passing-local-variables